modules/claims_api/app/controllers/concerns/claims_api/disability_compensation_validations.rb

```diff
@@ -58,8 +59,7 @@ def validate_form_526_current_mailing_address_country!
     end
 
     def valid_countries
-      @current_user.last_signed_in = Time.now.iso8601 if @current_user.last_signed_in.blank?
-      @valid_countries ||= EVSS::ReferenceData::Service.new(@current_user).get_countries.countries
```

Comment:
I noticed the method retrieve_separation_locations in the same class also uses the EVSS ReferenceData service (EVSS::ReferenceData::Service.new(@current_user).get_separation_locations). Not sure if that data is available in BRD as well, but if so should we move that over at this point as well?